src/pages/index.tsx

```diff
@@ -44,6 +44,8 @@ const IndexPage = ({
     for (let i = 0; i < allProfiles.length; i++) {
       const fellow = new Fellow(
         allProfiles[i].frontmatter as FellowType,
+        allProfiles[i].body,
+        //  allProfiles[i].fields.slug,
```

Comment:
I'm trying to pass slug (aka the link to the PortfolioPage) into `Fellow` so that `PortfolioModal` can access it. I'm getting a type error when I add it to the graphql query at the bottom of this file, so I just commented out the slug for now.